### PR TITLE
Fix IE 9 & 10 design bugs

### PR DESCRIPTION
**Pull request by blazepollard**
updated on IE9/10:
screen shot 2017-05-01 at 3 38 19 pm

### CODE REVIEW

File: static/canjs.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+var flexibility = require("flexibility");
```
Comment (chasenlehara) [inline static/canjs.js]: @imaustink, could you update this PR to conditionally load this module for only IE 9 & 10? I think [steal-conditional’s boolean conditionals](https://stealjs.com/docs/steal-conditional.html#boolean-conditionals) would be a good fit for this, but I could be wrong.

File: static/needs-flexibility.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    else  // If another browser, return 0
+    {
+        return 0;
+    }
+    
+}
+
+var v = msieversion();
+
+export default (v >= 9 && v <= 10);
```
Comment (chasenlehara) [inline static/needs-flexibility.js]: Instead of testing for IE 9 and 10 specifically, you might want to test for flexbox support.

Comment (imaustink) [inline static/needs-flexibility.js]: Re you thinking we use something like [Modernizr](https://modernizr.com/docs/#features-css)? I am not finding a good way of testing flexbox supper besides using Modernizr.

Comment (justinbmeyer) [inline static/needs-flexibility.js]: How does modernized test it?

Comment (imaustink) [inline static/needs-flexibility.js]: @justinbmeyer I downloaded a build of Modernizr that only includes flexbox detector. The only way to download it was minified so I ran it through JSNice: [gist](https://gist.github.com/imaustink/9e256d92935b315512b1000fcc51b7dd)

**Comment by imaustink** [comment]
@chasenlehara this is ready for review again. I have updated it to use steal-conditional to load `flexibility`. Let me know if you have any thoughts on testing flexbox support. I am open to ideas if you think it's important.

File: static/header.less [hunk 3]
```diff
@@ -112,7 +111,7 @@
 .top-right-links {
     display: none;
     @media screen and (min-width: @breakpoint){
-        display: flex;
```
Comment (chasenlehara) [inline static/header.less]: Was this supposed to be removed?

File: static/layout.less [hunk 4]
```diff
@@ -1,4 +1,5 @@
 #everything {
+  -js-display: flex; // ie9/10 flexbox fix
```
Comment (chasenlehara) [inline static/layout.less]: Could this be added to the `.flex` mixin?

File: static/needs-flexibility.js (hunk 2 above)
Comment (chasenlehara) [inline static/needs-flexibility.js]: https://github.com/Modernizr/Modernizr/search?utf8=✓&q=flexbox

This still needs to be addressed.

File: static/header.less (hunk 3 above)
Comment (imaustink) [reply static/header.less]: Yes, it's now applied by the `.flex` mixin instead.

File: static/layout.less (hunk 4 above)
Comment (imaustink) [reply static/layout.less]: Great idea. Done!

**Comment by imaustink** [comment]
@chasenlehara how does it look now?

File: package.json [hunk 5]
```diff
@@ -30,26 +30,28 @@
     "url": "https://github.com/canjs/bit-docs-html-canjs/issues"
   },
   "homepage": "https://github.com/canjs/bit-docs-html-canjs#readme",
-  "system": {},
```
Comment (chasenlehara) [inline package.json]: Was this configuration not required? https://stealjs.com/docs/StealJS.guides.substitution_conditional_loading.html#set-up-steal-conditional

**Comment by chasenlehara** [comment]
@imaustink Seems ok, would be good to get screenshots in IE 9 & 10 as final verification. I think @blazepollard was using a service that makes this easy.

**Comment by imaustink** [comment]
Not pretty, but useable in IE 10:
image

**Pull request closed without merging**